### PR TITLE
Miscellaneous fixes based on code review

### PR DESCRIPTION
**Pull request by GBuella**
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmemfile/145)

### CODE REVIEW

File: src/libpmemfile/preload.c [hunk 1]
```diff
@@ -713,217 +731,220 @@ dispatch_syscall(long syscall_number,
...
 		return hook_faccessat(cwd_desc(), arg0, 0);
 
-	if (syscall_number == SYS_faccessat)
+	case SYS_faccessat:
 		return hook_faccessat(fetch_fd(arg0), arg1, arg2);
 
 	/*
 	 * The newfstatat syscall implements both stat and lstat.
-	 * Linux calls it: newfstatat ( I guess there was an old one )
+	 * Linux calls it: newfstatat ( I guess there was an old one :
```
Comment (marcinslusarz) [inline src/libpmemfile/preload.c]: ?

Comment (GBuella) [inline src/libpmemfile/preload.c]: Well, that is not the most useful change I ever made in my life.

Comment (GBuella) [inline src/libpmemfile/preload.c]: Fixed

**Comment by marcinslusarz** [comment]
:lgtm:

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/145#-:-Kkl2CYZtdKehPwrPkNK:bnfp4nl)*

**Pull request merged**